go/libraries/doltcore/sqle/enginetest/stats_queries.go

```diff
@@ -569,6 +569,73 @@ var DoltStatsIOTests = []queries.ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "differentiate table cases",
+		SetUpScript: []string{
+			"set @@PERSIST.dolt_stats_auto_refresh_interval = 0;",
+			"set @@PERSIST.dolt_stats_auto_refresh_threshold = 0;",
+			"set @@PERSIST.dolt_stats_branches ='main'",
+			"CREATE table XY (x bigint primary key, y varchar(16))",
+			"insert into XY values (0,'0'), (1,'1'), (2,'2')",
+			"analyze table XY",
+		},
+		Assertions: []queries.ScriptTestAssertion{
+			{
+				Query:    "select table_name, upper_bound from dolt_statistics",
+				Expected: []sql.Row{{"xy", "2"}},
+			},
+		},
+	},
+	{
+		Name: "deleted table loads OK",
+		SetUpScript: []string{
+			"set @@PERSIST.dolt_stats_auto_refresh_interval = 0;",
+			"set @@PERSIST.dolt_stats_auto_refresh_threshold = 0;",
+			"set @@PERSIST.dolt_stats_branches ='main'",
+			"CREATE table xy (x bigint primary key, y varchar(16))",
+			"insert into xy values (0,'0'), (1,'1'), (2,'2')",
+			"analyze table xy",
+			"CREATE table uv (u bigint primary key, v varchar(16))",
+			"insert into uv values (0,'0'), (1,'1'), (2,'2')",
+			"analyze table uv",
+			"drop table uv",
+		},
+		Assertions: []queries.ScriptTestAssertion{
+			{
+				Query:    "select table_name, upper_bound from dolt_statistics",
+				Expected: []sql.Row{{"xy", "2"}},
+			},
+		},
+	},
+	{
+		Name: "differentiate branch cases",
```

Comment:
I don't see anything about cases in this test?

Reply:
renamed test, confirmed that branch names are case-insensitive